app/services/food_service.rb

```diff
@@ -0,0 +1,20 @@
+# app/services/food_service.rb
+
+class FoodService 
+  def self.connection
+    Faraday.new(url: 'https://api.nal.usda.gov/fdc/v1/') do |faraday|
+      faraday.params['api_key'] = ENV['food_api_key']
+    end
+  end
+
+  def self.foods_containing_ingredient(query)
+    response = connection.get('foods/search') do |request|
+      request.params['api_key'] = ENV['food_api_key']
```

Comment:
We can remove this line because you have already set your API key in the `connection` method above.